src/components/Quiz/QuizzesStats.tsx

```diff
@@ -0,0 +1,192 @@
+import React, { useContext } from "react"
+import {
+  Box,
+  Circle,
+  Flex,
+  Grid,
+  GridItem,
+  Icon,
+  Progress,
+  Stack,
+  Text,
+} from "@chakra-ui/react"
+import { FaTwitter } from "react-icons/fa"
+
+import Button from "../Button"
+import Translation from "../Translation"
+import { TrophyIcon } from "../icons/quiz"
+
+import { QuizzesHubContext } from "./context"
+
+import {
+  getNumberOfCompletedQuizzes,
+  getTotalQuizzesPoints,
+  shareOnTwitter,
+} from "./utils"
+import { trackCustomEvent } from "../../utils/matomo"
+
+import { QuizShareStats } from "../../types"
+
+import { ethereumBasicsQuizzes, usingEthereumQuizzes } from "../../data/quizzes"
+
+const handleShare = ({ score, total }: QuizShareStats) => {
+  shareOnTwitter({
+    score,
+    total,
+  })
+
+  trackCustomEvent({
+    eventCategory: "quiz_hub_events",
+    eventAction: "Secondary button clicks",
+    eventName: "Twitter_share_stats",
+  })
+}
+
+const QuizzesStats: React.FC = () => {
+  const { score: userScore, completed, average } = useContext(QuizzesHubContext)
+  const totalQuizzesNumber =
+    ethereumBasicsQuizzes.length + usingEthereumQuizzes.length
+  const TOTAL_QUIZZES_POINTS = getTotalQuizzesPoints()
+  const numberOfCompletedQuizzes = getNumberOfCompletedQuizzes(
+    JSON.parse(completed)
+  )
+
+  const computedAverage =
+    average.length > 0 ? average.reduce((a, b) => a + b, 0) / average.length : 0
+  const parsedAverage = Number.isInteger(computedAverage)
+    ? computedAverage
+    : computedAverage.toFixed(2)
+
+  return (
+    <Box flex={1} order={{ base: 1, lg: 2 }} w="full">
+      <Stack mt={{ base: 0, lg: 12 }} gap={{ base: 8, lg: 4 }}>
+        {/* user stats */}
+        <Grid
+          gap={4}
+          bg="backgroundHighlight"
+          borderRadius={{ base: "none", lg: "lg" }}
+          border="none"
+          p={4}
```

Comment:
```suggestion
          p={8}
```

Fixes alignment 

<img width="429" alt="Screenshot 2023-06-02 at 11 32 02" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/ea71e4d5-6c6e-4279-95be-7c41ccd86d06">
<img width="366" alt="Screenshot 2023-06-02 at 11 36 11" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/8f70b95e-70ef-458a-bd11-2516baa7ff61">

Reply:
@nloureiro defined to use 16px padding on all sizes

Reply:
@nloureiro or @konopkja any thoughts here?

The spacing on Figma is pretty inconsistent for this, so I don't really know what value is correct when I'm reviewing it. Could we please update Figma to reflect the final designs 🙏

Reply:
This is more of a design decision, but I miss the space between the hero and the content.

On Figma, this space is not there, maybe on purpose, @konopkja was this intentional?
[figma screeshot]
![Screen Shot 2023-06-02 03 51 58 PM](https://github.com/ethereum/ethereum-org-website/assets/1120748/f5d0c5c4-9725-4f01-a6cb-6c512003be5b)



Here I added 32px in the dev tools to demonstrate the difference.
https://github.com/ethereum/ethereum-org-website/assets/1120748/b08ec273-49b6-43f8-a305-33439abe0afb

Reply:
if the discussion is about the empty space between the "header" and "total points" box then yes, it was intentional to not have it there. 

That however assumed that the padding between "Your total points" and the header is 32px

Reply:
we can either: 

1. keep the 16px padding and introduce another empty space between the header and the box

or

2. keep the box sticked to the header, but then make the padding for "Your total points" 32, not 16

Reply:
@nloureiro @konopkja I'm talking about the alignment of items vs gutters

<img width="358" alt="Screenshot 2023-06-02 at 16 05 41" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/ad005793-3cb0-4bf7-8b54-498d13e54a97">

Reply:
IMO should be `p={{base: 8, lg: 4}}`, 2rem (32px) for mobile and 1rem (16px) for desktop, as Jakub suggested before more padding works better for smaller devices

Reply:
ok I see, can you do do it 32px then

![Screen Shot 2023-06-05 03 12 53 PM](https://github.com/ethereum/ethereum-org-website/assets/1120748/39b8cbd3-65d2-4fc9-b91b-1bb25a1d9b50)